### PR TITLE
Make OrderedBy not exported

### PR DESCRIPTION
**Pull request by szepeviktor**
- [x] The contributed code is licensed under GPL v2.0
- [x] Contributor Licence Agreement (CLA) is signed
- [ ] util/update-modules has been ran
- [ ] Documentation updated
- [ ] Test suite update

Make OrderedBy's return type exported as OrderedBy is exported.

### CODE REVIEW

**Comment by svetasmirnova** [comment]
What is the reason of this change? main.go is not library code. Where are you going to use this structure?

**Comment by szepeviktor** [comment]
I've just discovered
```go
func OrderedBy(less ...lessFunc) *multiSorter {
```
that an exported function has a not exported return type.

File: src/go/pt-mongodb-query-digest/main.go [hunk 1]
```diff
@@ -407,32 +407,32 @@ func getTotalsTemplate() string {
...
+func (ms *MultiSorter) Sort(queries []stats.QueryStats) {
 	ms.queries = queries
 	sort.Sort(ms)
 }
 
 // OrderedBy returns a Sorter that sorts using the less functions, in order.
 // Call its Sort method to sort the data.
-func OrderedBy(less ...lessFunc) *multiSorter {
-	return &multiSorter{
+func OrderedBy(less ...lessFunc) *MultiSorter {
```
Comment (svetasmirnova) [inline src/go/pt-mongodb-query-digest/main.go]: Sort, Len, Swap, Less are methods of the sort interface: https://pkg.go.dev/sort#Interface I suggest to leave name multiSorter as is and rename OrderedBy instead:

```suggestion
func orderedBy(less ...lessFunc) *multiSorter {
```

**Review by svetasmirnova — CHANGES_REQUESTED** [review]

**Comment by svetasmirnova** [comment]
Also found this sample: https://devbits.app/x/761/using-sort-sortmultikeys-in-go

File: src/go/pt-mongodb-query-digest/main.go (hunk 1 above)
Comment (szepeviktor) [reply src/go/pt-mongodb-query-digest/main.go]: Done 🍏

**Review by artemgavrilov — APPROVED** [review]

**Review by svetasmirnova — APPROVED** [review]

**Pull request merged**